### PR TITLE
Feature: Settings page edit

### LINKED ISSUE #94: [Feature} : Creating the Settings page of the website
A settings page is need to control the profile , theme , user authentication of the website. This page will contain the most crucial part of the user.

I would like to work on this under GSSoC'24

### PR DESCRIPTION
**Pull request by AkhilaSunesh**
### Related Issue(s)

- Fixes #94 #129

### Description

Added a settings page
image

### Checklist

Before submitting your pull request, ensure that you have completed the following tasks:

- [x] Code follows the project's style and conventions.
- [x] Code has been tested thoroughly, and all tests pass successfully.
- [ ] Documentation has been updated to reflect any relevant changes.
- [ ] All new and modified code is adequately commented for clarity.
- [x] Pull request title follows the specified format: [Feature/Enhancement/Bug Fix]: Brief description of the change.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.6.0",
```
Comment (Swarnendu0123) [inline client/package.json:13]: What is the use case of this dependency?

File: client/src/components/pages/settings.tsx [hunk 2]
```diff
@@ -1,9 +1,139 @@
+import React from 'react';
+
 const Settings = () => {
-    return (
-        <div className="bg-[#f4f6fa] w-full h-full">
-            <h1>This is a Settings page</h1>
+  return (
```
Comment (Swarnendu0123) [inline client/src/components/pages/settings.tsx:4]: Use `shadcn ui` instead of normal components.

File: client/src/components/ui/form.tsx [hunk 3]
```diff
@@ -0,0 +1,176 @@
+import * as React from "react"
+import * as LabelPrimitive from "@radix-ui/react-label"
```
Comment (Swarnendu0123) [inline client/src/components/ui/form.tsx:2]: Use `snadcn ui` here also.

**Comment by Swarnendu0123** [comment]
Overall good but you added tabs like, work opportunities, application and more. How these tabs will be helpful?

**Comment by AkhilaSunesh** [comment]
I have followed the template as such in issue #94....can I make it more practical? Like removing some tabs and all...

You had mentioned that you wanted the exact thing as the picture. That why I tried to incorporate it as much as I could

**Comment by Swarnendu0123** [comment]
Actually I meant to say, you have created tw PRs on a same issue. We need to accept only ine of them.

**Comment by AkhilaSunesh** [comment]
> Actually I meant to say, you have created two PRs on a same issue. We need to accept only ine of them.

Ok, so I guess you want this template right? unlike the previous one

**Comment by Swarnendu0123** [comment]
Thanks for your pull request. But for now we don't have a good design of the settings page. Feel free to create another if you have something new.

**Pull request closed without merging**